jenkins.yaml

```
@@ -1,6 +1,6 @@
 jenkins:
   agentProtocols:
-  - "JNLP2-connect"
+  # - "JNLP2-connect"
```

Either remove it or add a comment. I added protocol to showcase that you can override Jenkins defaults, this demo will indeed showing a security warning. But this is fine with me

```suggestion
  # Demonstrates that a disabled protocol can be enabled via JCasC. Note that it will lead to the administrative warning
  - "JNLP2-connect"
```